pkgs/by-name/hd/hdrop/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  scdoc,
+  coreutils,
+  util-linux,
+  jq,
+  libnotify,
+  hyprland ? null,
```

Never use null as default value.

```suggestion
  withHyprland ? true, hyprland
```